lib/api/routes.js

```diff
@@ -7,4 +7,16 @@ module.exports = [
     {
         path: '/_/healthcheck',
     },
+    {
+        path: '/_/backlog',
```

Comment:
With the current setup, I wasn't sure how best to setup a prepended "metrics" route. So currently my `routes.js` would look something like:
`{ path: '/_/metrics/backlog',  method: 'getBacklog' }`

Now instead of relying on the path and parsing it to determine which method to call, I will now match by path names and I use a predefined method for calling the right method